modules/node_groups/locals.tf

```diff
@@ -26,6 +26,7 @@ locals {
       taints                        = []
       update_default_version        = true
       ebs_optimized                 = null
+      update_config                 = tomap({ "max_unavailable" : 1 })
```

Comment:
I am not sure that this is the current behavior for existing users (also, please remove `tomap()`).

The goal is to introduce new features into the module without breaking anything or triggering updates to existing infrastructure for current users.

This should be verified like this:
1. Bring cluster up using the code in `examples` from `master` before this PR
2. Add the changes from this PR
3. Run `terraform plan` and there should be no changes proposed (especially, no destructive changes)

Please provide the final `terraform plan`.

Reply:
@antonbabenko  done. Please find attached two plans.
1) As you requested the post-change plan overt the infra created pre-change showing there is no change
2) A post-change plan using the parameters I added to the example showing that update_config will be applied

File: [tfplans.zip](https://github.com/terraform-aws-modules/terraform-aws-eks/files/7098455/tfplans.zip)